index.js

```diff
@@ -37,6 +49,12 @@ const useApi = ({
       return;
     }
 
+    if (changedRef.current && typeof changedRef.current !== "function") {
+      setLoading(false);
+      setError("Invalid changed type, must be function.");
```

Comment:
Hmmm... same here...

Maybe we should be **throwing** errors if its a developer error rather than network error or similar?